### PR TITLE
For  #7970 - added search metrics to baseline ping

### PR DESCRIPTION
**Pull request by perrymcmanis144**
### Pull Request checklist

- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

Tests: this counter is already sent in metrics & is covered by existing tests
Screenshots: no visible changes
Accessibility: does not impact any user facing feature

### QA

- [ ] **QA Needed**

### To download an APK when reviewing a PR:
1. click on `Show All Checks`,
2. click `Details` next to `build-focus-debug` or `build-klar-debug` for changes targeting Klar,
2. click `View task in Taskcluster`,
3. click the `Artifacts` row,
4. click to download any of the apks listed here which use an appropriate name for each CPU architecture.

### GitHub Automation

Used by GitHub Actions.

### CODE REVIEW

File: app/metrics.yaml [hunk 1]
```diff
@@ -1331,7 +1350,7 @@ search_engines:
       - rtestard@mozilla.com
     expires: never
     extra_keys:
-      current_engines_count:
+      current_:
```
Comment (Dexterp37) [inline app/metrics.yaml:1353]: @perrymcmanis144 is this intentional?

Comment (perrymcmanis144) [reply app/metrics.yaml:1353]: I don't have this in my local...

ok let me fix

File: app/metrics.yaml [hunk 2]
```diff
@@ -1350,7 +1350,7 @@ search_engines:
...
```
Comment (perrymcmanis144) [inline app/metrics.yaml:1353]:
```suggestion
      current_engines_count:
```

**Review by Dexterp37 — APPROVED** [review]

**Review by jonalmeida — APPROVED** [review]
This sounds straight-forward from what I've read in the issue description. ✅ 

@perrymcmanis144 do we want this uplifted to a beta/release build?

**Comment by perrymcmanis144** [comment]
@jonalmeida thanks for the review and yes, it would be greatly appreciated to get this uplifted.

**Comment by rocketsroger** [comment]
@perrymcmanis144 Updated to Beta v107.  Thanks

**Pull request merged**